src/ORM/Marshaller.php

```diff
@@ -106,13 +106,19 @@ protected function _buildPropertyMap(array $data, array $options): array
                 $nested['forceNew'] = $options['forceNew'];
             }
             if (isset($options['isMerge'])) {
-                $callback = function ($value, EntityInterface $entity) use ($assoc, $nested) {
+                $callback = function (
+                    $value,
+                    EntityInterface $entity
+                ) use (
+                    $assoc,
+                    $nested
+                ): array|EntityInterface|null {
```

Comment:
i think this formatting can be improved

Reply:
How? Any different and CS fails (too long line length) etc.

Reply:
I thought maybe we can put parts of that function definition in 1 line but it seems we can't...